pkgs/by-name/se/searxng/package.nix

```diff
@@ -5,13 +5,13 @@
 
 python3.pkgs.toPythonModule (python3.pkgs.buildPythonApplication rec {
   pname = "searxng";
-  version = "0-unstable-2024-02-24";
+  version = "0-unstable-2024-02-26";
 
   src = fetchFromGitHub {
     owner = "searxng";
     repo = "searxng";
-    rev = "d72fa99bd0a4d702a55188b07919ce5a764b1d6c";
-    hash = "sha256-1A7dyWrF63fSSvWP+2HrCS6H8o/4CUlqiP0KANVZHUA=";
+    rev = "22ffeb2702403e352ea4266f1b1c8b32a82e4564";
```

Comment:
This is the PR commit from a fork, not the actual searxng repo. While this works, is there any reason why we should use this commit instead of anything from `master` after the PR was merged?

Reply:
Good catch, we shouldn't